### PR TITLE
New Memecoin for VeChain

### PR DESCRIPTION
**Pull request by Murtagh300**

### CODE REVIEW

**Comment by Murtagh300** [comment]
@XJWX89 Please look into this

File: tokens/main/0xb27a1fb87935b85cdaa2e16468247278c74c5ec7/additional.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "website":"No Website",
```
Comment (libotony) [inline tokens/main/0xb27a1fb87935b85cdaa2e16468247278c74c5ec7/additional.json:2]: This line should be removed.

File: tokens/test/0xcb384a3a82bfe0619963d65a05f3801ddc23f3cc/additional.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "website":"No Website",
```
Comment (libotony) [inline tokens/test/0xcb384a3a82bfe0619963d65a05f3801ddc23f3cc/additional.json:2]: This line should be removed.

**Review by libotony — CHANGES_REQUESTED** [review]

**Comment by Murtagh300** [comment]
Should be fixxed now! @libotony

**Review by libotony — APPROVED** [review]

File: tokens/test/0xcb384a3a82bfe0619963d65a05f3801ddc23f3cc/Logo.png
Comment (libotony) [inline tokens/test/0xcb384a3a82bfe0619963d65a05f3801ddc23f3cc/Logo.png:1]: This file should be named "logo.png"

**Review by libotony — CHANGES_REQUESTED** [review]

**Comment by Murtagh300** [comment]
@libotony Changed both Logos in Correct logo.png Format

**Comment by libotony** [comment]
> @libotony Changed both Logos in Correct logo.png Format

Sorry, I made a mistake, the name should be 'token.png'.

**Comment by Murtagh300** [comment]
Fixxed @libotony ❤️❤️

**Review by libotony — APPROVED** [review]

**Pull request merged**